src/styles/Mixin.js

```diff
@@ -0,0 +1,7 @@
+import { css } from 'styled-components';
+
+export const Flex = (flex, none, nomal) => css`
```

Comment:
```suggestion
export const flex = (flex, none, nomal) => css`
```
대문자로 시작하는건 컴포넌트 이름입니다. 함수 등 변수명은 소문자로 시작해주세요

Reply:
수정했습니다~